daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DarReader.scala

```diff
@@ -98,6 +109,7 @@ object Errors {
 object DarReader {
 
   private val ManifestName = "META-INF/MANIFEST.MF"
+  private val EntrySizeThreshold = 1024 * 1024 * 1024 // 1 GB
```

Comment:
If you simply injected this as a constructor parameter you could simplify your testing efforts immensely (i.e., you would not need to generate a real zip bomb using a Python script plus you could run the test case on all platforms as a simple unit test).

Reply:
Good point, thanks.